src/main/java/org/dinosaur/foodbowl/domain/review/presentation/ReviewControllerDocs.java

```diff
@@ -0,0 +1,39 @@
+package org.dinosaur.foodbowl.domain.review.presentation;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Size;
+import java.util.List;
+import org.dinosaur.foodbowl.domain.member.domain.Member;
+import org.dinosaur.foodbowl.domain.review.dto.request.ReviewCreateRequest;
+import org.dinosaur.foodbowl.global.exception.ExceptionResponse;
+import org.dinosaur.foodbowl.global.presentation.Auth;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestPart;
+import org.springframework.web.multipart.MultipartFile;
+
+@Tag(name = "리뷰", description = "리뷰 API")
+public interface ReviewControllerDocs {
+
+    @Operation(summary = "리뷰 등록", description = "가게에 해당하는 리뷰를 등록합니다.")
+    @ApiResponses({
+            @ApiResponse(responseCode = "201", description = "리뷰 등록 성공"),
+            @ApiResponse(
+                    responseCode = "400",
+                    description = "사진이 4개보다 많은 경우, 가게 이름 값이 없는 경우, 가게 주소 필드가 값이 없는 경우, "
+                            + "가게 경도 값이 없는 경우, 가게 위도 값이 없는 경우, 가게 정보 URL이 없는 경우, "
+                            + "가게 카테고리 정보가 없는 경우, 가게 리뷰 내용이 없는 경우",
```

Comment:
모든 예외 상황에 대해서 꼼꼼히 작성해주셨네요 👍
```suggestion
                    description = "1.사진이 4개보다 많은 경우\t\n2.가게 이름 값이 없는 경우\t\n3.가게 주소 필드가 값이 없는 경우"
                            + "\t\n4.가게 경도 값이 없는 경우\t\n5.가게 위도 값이 없는 경우\t\n6.가게 정보 URL이 없는 경우"
                            + "\t\n7.카테고리 정보가 없는 경우\t\n8.가게 리뷰 내용이 없는 경우",
```
<img width="1310" alt="스크린샷 2023-08-15 오전 1 24 47" src="https://github.com/Busan-Dinosaur/Foodbowl-Backend/assets/78212016/813f8192-539b-4df2-bfe2-bd95a67fded6">

<img width="331" alt="스크린샷 2023-08-15 오전 1 24 18" src="https://github.com/Busan-Dinosaur/Foodbowl-Backend/assets/78212016/849fcc67-f3f5-4288-a3e0-68175251ed87">

숫자로 가독성을 챙겨보는건 어떨까요?! 😀

숫자와 내용 사이에 공백을 주게 되면 `ordered list`로 인식하게 되서 추가적인 공백이 생기게 되서 숫자와 내용 사이는 붙이도록 하였습니다!